### ISSUE TITLE
Dist Build failed ford v1.11

### ISSUE THREAD

**Issue by BrookeDot** [Kovah/LinkAce#587]
### Bug Description

The build failed for v1.11:
https://github.com/Kovah/LinkAce/actions/runs/3702519408

```
Your lock file does not contain a compatible set of packages. Please run composer update.

  Problem 1
    - Root composer.json requires php ^8.0.2 | ^8.1 | ^8.2 but your php version (7.4.33) does not satisfy that requirement.
  Problem 2
    - laravel/framework is locked to version v9.43.0 and an update of this package was not requested.
    - laravel/framework v9.43.0 requires php ^8.0.2 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 3
    - league/flysystem is locked to version 3.11.0 and an update of this package was not requested.
    - league/flysystem 3.11.0 requires php ^8.0.2 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 4
    - league/flysystem-aws-s3-v3 is locked to version 3.10.3 and an update of this package was not requested.
    - league/flysystem-aws-s3-v3 3.10.3 requires php ^8.0.2 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 5
    - league/flysystem-ftp is locked to version 3.10.3 and an update of this package was not requested.
    - league/flysystem-ftp 3.10.3 requires php ^8.0.2 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 6
    - league/flysystem-sftp-v3 is locked to version 3.10.3 and an update of this package was not requested.
    - league/flysystem-sftp-v3 3.10.3 requires php ^8.0.2 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 7
    - nunomaduro/termwind is locked to version v1.14.2 and an update of this package was not requested.
    - nunomaduro/termwind v1.14.2 requires php ^8.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 8
    - psr/simple-cache is locked to version 3.0.0 and an update of this package was not requested.
    - psr/simple-cache 3.0.0 requires php >=8.0.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 9
    - ramsey/uuid is locked to version 4.6.0 and an update of this package was not requested.
    - ramsey/uuid 4.6.0 requires php ^8.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 10
    - spatie/db-dumper is locked to version 3.3.0 and an update of this package was not requested.
    - spatie/db-dumper 3.3.0 requires php ^8.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 11
    - spatie/laravel-backup is locked to version 8.1.5 and an update of this package was not requested.
    - spatie/laravel-backup 8.1.5 requires php ^8.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 12
    - spatie/laravel-package-tools is locked to version 1.13.7 and an update of this package was not requested.
    - spatie/laravel-package-tools 1.13.7 requires php ^8.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 13
    - spatie/laravel-signal-aware-command is locked to version 1.2.0 and an update of this package was not requested.
    - spatie/laravel-signal-aware-command 1.2.0 requires php ^8.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 14
    - spatie/temporary-directory is locked to version 2.1.1 and an update of this package was not requested.
    - spatie/temporary-directory 2.1.1 requires php ^8.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 15
    - symfony/console is locked to version v6.0.16 and an update of this package was not requested.
    - symfony/console v6.0.16 requires php >=8.0.2 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 16
    - symfony/css-selector is locked to version v6.0.11 and an update of this package was not requested.
    - symfony/css-selector v6.0.11 requires php >=8.0.2 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 17
    - symfony/deprecation-contracts is locked to version v3.0.2 and an update of this package was not requested.
    - symfony/deprecation-contracts v3.0.2 requires php >=8.0.2 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 18
    - symfony/error-handler is locked to version v6.0.15 and an update of this p
```

### How to reproduce

Visit the assets page and notice that there is not a 1.11 dist build available

As a workaround one can manually run `composer update` from the source build.

### Expected behavior

Have a build that includes all assets.

### LinkAce version

v1.11

### Setup Method

PHP

### Operating System

Linux (Ubuntu, CentOS,...)

**Comment by Kovah** [comment]
> your php version (7.4.33) does not satisfy that requirement

LinkAce does not support PHP 7.4 anymore. It only supports the latest version that receives at least security fixes, which is 8.0 at the moment.

**Comment by BrookeDot** [comment]
Thanks, I actually meant the official build failed, but see you fixed it in 5aef7c5c1928f93a456a976a51ad4621e9684849 so closing this out :D
